pyke/lightcurve.py

```diff
@@ -93,12 +93,40 @@ def flatten(self, window_length=101, polyorder=3, **kwargs):
     def fold(self, phase, period):
         return LightCurve(((self.time - phase + 0.5 * period) / period) % 1 - 0.5, self.flux)
 
-    def draw(self):
-        raise NotImplementedError("Should we implement a LightCurveDrawer class?")
-
     def to_csv(self):
         raise NotImplementedError()
 
+    def plot(self, t=None, flux=None, flux_err=None, ax=None, norm=True, xlabel = 'Time - 2454833 (days)',
+            ylabel = 'Normalised Flux', title=None, color='#363636', fill=False, grid=True, legend=True, **kwargs):
+        """
+        Plot a Light Curve.
+        """
+        if ax is None:
+            fig, ax = plt.subplots(1)
+        if t == None:
+            t = self.time
+        if flux == None:
+            flux = self.flux
+        if flux_err == None:
+            flux_err = self.flux_err
+        if norm:
+            flux_err/=np.nanmedian(flux)
+            flux/=np.nanmedian(flux)
+        plt.errorbar(t, flux,flux_err, color=color, **kwargs)
+        if fill:
+            plt.fill(t, flux, fc='#a8a7a7', linewidth=0.0, alpha=0.3)
+        if grid:
+            plt.grid(alpha=0.3)
+        if legend:
+            plt.legend()
+        if not( title is None):
+            plt.title(title)
+        plt.xlabel(xlabel, {'color' : 'k'})
+        plt.ylabel(ylabel, {'color' : 'k'})
+
```

Comment:
Let's add `return ax` here, so the user can edit the plot further if needed.

Reply:
👍